packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -0,0 +1,189 @@
+};
+
+function Environment(props) {
+	return (
+		<li className="tc-header-bar-action">
+			<ActionDropdown
+				bsStyle="link"
+				icon="talend-burger"
+				id={props.id}
+				items={props.items}
+				label={props.label}
+				tooltipPlacement="bottom"
+			/>
+		</li>
+	);
+}
+
+Environment.propTypes = {
+	id: React.PropTypes.string,
+	items: ActionDropdown.propTypes.items,
+	label: ActionDropdown.propTypes.label,
+};
+
+function Search(props) {
+	return (
+		<li className="tc-header-bar-action separated flex">
+			<Typeahead {...props} />
+		</li>
+	);
+}
+
+Search.propTypes = Typeahead.propTypes;
+
+function Help({ id, onClick }) {
+	return (
+		<li className="tc-header-bar-action">
+			<Action
+				bsStyle="link"
+				icon="talend-question-circle"
+				id={id}
+				label="Help"
+				onClick={onClick}
+				tooltipPlacement="bottom"
+			/>
+		</li>
+	);
+}
+
+Help.propTypes = {
+	id: React.PropTypes.string,
+	onClick: React.PropTypes.func.isRequired,
+};
+
+function User({ id, items, name }) {
+	return (
+		<li className="tc-header-bar-action separated">
+			<ActionDropdown
+				bsStyle="link"
+				icon="talend-user-circle"
+				id={id}
+				items={items}
+				label={name}
+				noCaret
+				tooltipPlacement="bottom"
+			/>
+		</li>
+	);
+}
+
+User.propTypes = {
+	id: React.PropTypes.string,
+	items: ActionDropdown.propTypes.items,
+	name: ActionDropdown.propTypes.label,
+};
+
+function Products({ id, items }) {
+	return (
+		<li className="tc-header-bar-action">
+			<ActionDropdown
+				bsStyle="link"
+				id={id}
+				icon="talend-launcher"
+				items={items}
+				label="Apps"
+				noCaret
+				pullRight
+				tooltipPlacement="bottom"
+			/>
+		</li>
+	);
+}
+
+Products.propTypes = {
+	id: React.PropTypes.string,
+	items: ActionDropdown.propTypes.items,
+};
+
+function HeaderBar(props) {
+	return (
+		<nav className={classNames([`tc-header-bar, ${theme['tc-header-bar']}`])} >
+			<ul className="tc-header-bar-actions">
+				<Logo {...props.logo} />
+				<Brand {...props.brand} isSeparated={!!props.env} />
+				{props.env && (<Environment {...props.env} />)}
+			</ul>
+			<ul className="tc-header-bar-actions right">
+				{props.search && (<Search {...props.search} />)}
+				<Help {...props.help} />
+				<User {...props.user} />
+				<Products {...props.products} />
+			</ul>
+		</nav>
+	);
+}
+
+HeaderBar.Logo = Logo;
+HeaderBar.Brand = Brand;
+HeaderBar.Environment = Environment;
+HeaderBar.Search = Search;
+HeaderBar.Help = Help;
+HeaderBar.User = User;
+HeaderBar.Products = Products;
+
+HeaderBar.propTypes = {
+	brand: React.PropTypes.shape(Brand.propTypes).isRequired,
+	logo: React.PropTypes.shape(Logo.propTypes).isRequired,
+	env: React.PropTypes.shape(Environment.propTypes),
+	search: React.PropTypes.shape(Search.propTypes),
+	help: React.PropTypes.shape(Help.propTypes).isRequired,
+	user: React.PropTypes.shape(User.propTypes).isRequired,
+	products: React.PropTypes.shape(Products.propTypes).isRequired,
```

Comment:
nothing should be required I guess. Only the logo. I have screen with only logo in the header bar